assets/training/model_management/components/download_model/spec.yaml

```diff
@@ -17,6 +17,7 @@ command: >
   --model-source ${{inputs.model_source}}
   --model-id '${{inputs.model_id}}'
   $[[--update-existing-model ${{inputs.update_existing_model}}]]
+  $[[--validation-info ${{inputs.validation_info}}]]
```

Comment:
please also add in run_model_download.py

Reply:
ok